docs/library/graphing/area_chart.md

```diff
@@ -150,6 +165,37 @@ range_area_chart = """rx.area_chart(
                 rx.x_axis(data_key="day"), 
                 rx.y_axis(),
                 data=range_data)"""
+
+
+area_chart_example_with_state = """rx.area_chart(
+            rx.area(
+                data_key="uv",
+                stroke="#8884d8",
+                fill="#8884d8",
+                type_="natural",
+                on_click=AreaState.randomize_data,
+
+            ),
+            rx.area(
+                data_key="pv",
+                stroke="#82ca9d", 
+                fill="#82ca9d",
+                type_="natural",
+            ),
+            rx.x_axis(
+                data_key="name",
+            ),
+            rx.y_axis(), 
+            rx.legend(),
+            rx.cartesian_grid(
+                stroke_dasharray="3 3",
+            ),
+            data=AreaState.data,
+            width=1000,
```

Comment:
Should not set an explicite width/height here so it stays responsive

Reply:
changed